api/apistruct/struct.go

```diff
@@ -189,6 +201,24 @@ type StorageMinerStruct struct {
 	}
 }
 
+type WorkerStruct struct {
+	Internal struct {
+		// TODO: lower perms
+
+		Version func(context.Context) (build.Version, error) `perm:"admin"`
+
+		TaskTypes func(context.Context) (map[sealtasks.TaskType]struct{}, error) `perm:"admin"`
+		Paths     func(context.Context) ([]stores.StoragePath, error)            `perm:"admin"`
+		Info      func(context.Context) (api.WorkerInfo, error)                  `perm:"admin"`
+
```

Comment:
@magik6k Is it possible for workers to be capable of performing the initial `pledge-sector` operation? I have noticed that it messes with a miner's ability to compute epost as there is no way to limit the number of gorountines used by `pledge-sector` and it starves epost

Reply:
It's a fairly easy optimization, but I want to keep the initial thing minimal (If you can call this PR that, heh)